packages/e2e/cypress/integration/common/liveness.ts

```diff
@@ -0,0 +1,14 @@
+import { Then } from '@badeball/cypress-cucumber-preprocessor';
+import { escapeRegExp } from 'lodash';
+
+// the current value returned by the service is 15000,
+// the below value can be increased as needed
+const LIVENESS_TIMEOUT = 16000;
```

Comment:
Update per rekognition request
```suggestion
const LIVENESS_TIMEOUT = 60000;
```